Entities/AHuman.h

```diff
@@ -878,6 +878,18 @@ ClassInfoGetters
 	void SetThrowPrepTime(long newPrepTime) { m_ThrowPrepTime = newPrepTime; }
 
 
+	/// <summary>
+	/// Gets this AHuman's Stride Sound. Ownership is not transferred!
+	/// </summary>
+	/// <returns>The SoundContainer for this AHuman's Stride Sound.</returns>
+	SoundContainer* GetStrideSound() const { return m_StrideSound; }
+
+	/// <summary>
+	/// Sets this AHuman's Stride Sound.
+	/// </summary>
+	/// <param name="newSound">The new SoundContainer for this AHuman's Stride Sound</param>
```

Comment:
See comments in ACrab.h about `*` placement, method comment capitalization and `.`s at the end of method comments.